<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>461</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">FONT:28PX;FONT-WEIGHT:BOLD;COLOR:RED</string>
   </property>
   <property name="text">
    <string>MỜI BẠN NHẬP VÀO(100-1000)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>BẮT ĐẦU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>NGỪNG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>THOÁT</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
